redbot/core/bot.py

```diff
@@ -1305,6 +1305,25 @@ async def is_owner(self, user: Union[discord.User, discord.Member]) -> bool:
         """
         return user.id in self.owner_ids
 
+    async def get_invite_url(self) -> str:
+        """
+        Generates the invite URL for the bot.
+
+        Does not check if invites are public.
+
+        Returns
+        -------
+        str
+            Invite URL.
+        """
+        app_info = await self.bot.application_info()
+        data = await self.bot._config.all()
```

Comment:
We're already *in* the bot object.
```suggestion
        data = await self._config.all()
```